web/templates/blocks/hosts_table.html.tmpl

```diff
@@ -6,7 +6,9 @@
                 <th scope='col'>Name</th>
                 <th scope='col'>Address</th>
                 <th scope='col'>Cluster</th>
-                <th scope='col'>Tags</th>
+                <th scope='col'>Environment</th>
```

Comment:
I'm not sure about the order. In the sidebar, we have `Systems`, `Landscapes` and `Environments` top to down, maybe we should follow the same order here, from left to right